cmake/LLVMIRUtil.cmake

```diff
@@ -20,7 +20,7 @@ llvmir_setup()
 # public (client) interface macros/functions
 
 function(llvmir_attach_bc_target)
-  set(options)
+  set(options LINK_STATIC_LIB)
```

Comment:
I'd prefer a more suggestive name:

1) first it should be plural `_LIBS`
2) it doesn't follow the CMake typical convention for such flags (although there is no standard)
3) it is not linking we are doing, that is done by the original CMake rules; we are just attaching to targets.
4) Moreover, in this case we are attaching to *dependent* libraries

So, I'd suggest a name like `ATTACH_TO_DEPENDENT_STATIC_LIBS`; nice and verbose.

Reply:
Modified as suggested in the new commit